typings/html-renderer.d.ts

```diff
@@ -35,6 +35,21 @@ declare global {
     ReportUIFeatures: typeof _ReportUIFeatures;
     Util: typeof _Util;
   }
+
+  module LH {
+    export interface ReportResult extends Result {
```

Comment:
maybe add a comment that this is for the bastardized report json we create within the report?

Reply:
done